soc/nxp/imxrt/rt6xx/Kconfig.soc

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_IMXRT_6XX
+	bool
+	select SOC_FAMILY_IMXRT
+
+config SOC_SERIES
+	default "rt6xx" if SOC_SERIES_IMXRT_6XX
+
+config SOC_MIMXRT685S_CM33
+	bool
+	select SOC_SERIES_IMXRT_6XX
+
+config SOC
+	default "mimxrt685s" if SOC_MIMXRT685S_CM33
```

Comment:
`CONFIG_MCUX_CORE_SUFFIX` needs to be set to `_cm33` in this file, or the RT685 will fail to build